### DISCUSSION TITLE
how to export peak matrix as an xlsx

### DISCUSSION THREAD

**Discussion by qimiaonnegliguo** [GreenleafLab/ArchR#982] [Questions / Documentation]
Hello! I have some confusion about the peak matrix.

After using getAvailableMatrices(projFour) I get 
[1] "GeneScoreMatrix" "MotifMatrix"    
[3] "PeakMatrix"      "TileMatrix"

Is there any method to export this PeakMatrix as an xlsx? It seems getMatrixFromProject() is not an option.

Thanks for your time and help   : )

**Comment by rcorces** [comment]
Welcome to ArchR - please make sure to follow instructions when using our GitHub page. You have posted a Question to the Issues section but this should have been posted to Discussions. This is indicated when you open a new Issue.
![image](https://user-images.githubusercontent.com/10343046/129584405-5703c450-3e1a-4e24-b1ed-85efb074318e.png)

I am converting your Issue to a Discussions post.

**Comment by rcorces** [comment]
No - there is no way to export an excel file directly from ArchR. If you would like to do this, you will need to manipulate the `SummarizedExperiment` object that ArchR gives you. You will have to Google to figure out how to do that as it is not an ArchR question.

**Reply by qimiaonnegliguo** [reply]
Apologize for the mistake and thank you for your reply. So is there a way to read peak information from ArchrRproject, such as start, end?

**Reply by rcorces** [reply]
Please read the manual in full.
https://www.archrproject.com/bookdown/calling-peaks-w-macs2.html
